code/src/components/RemoveAll.js

```diff
@@ -3,23 +3,12 @@ import { useDispatch } from 'react-redux';
 import { todos } from 'reducers/todos';
 import styled from 'styled-components';
 
+import { Button } from 'styling/Button';
+
 const ButtonWrapper = styled.div`
 text-align:center;
 `
-const Button = styled.button`
-  color: #3f3f3f;
-  font-size: 18px;
-  margin: 6px;
-  padding: 6px;
-  border: 1.5px solid #3f3f3f;
-  border-radius: 1.5px;
-  text-align:center;
-    &:hover {
-      background-color:#b83b5e;
-      color:white;
-      border:white
-    }
-`
+
 export const RemoveAll = () => {
```

Comment:
It's amazing to me how you managed to create a 'remove all' button in such a simple way! 😍 If you ever wanted to continue on this button, you could import a dependency called swal and create an alert for the user before they decide on removing all tasks. Here is how I've done it: `import Swal from "sweetalert2";
//reference: https://sweetalert2.github.io/` and then you can create a function which contains a remove button:

`  const removeItems = () => {
    Swal.fire({
      title: "Do you want to remove all tasks?",
      text: "This action cannot be undone once all tasks are deleted",
      showCloseButton: true,
      showCancelButton: true,
      focusConfirm: false,
      confirmButtonText: "yes, remove all",
      confirmButtonColor: "#93dece",
      confirmButtonAriaLabel: "yes",
      cancelButtonText: "no, keep tasks",
      cancelButtonColor: "#e18ea0",
      cancelButtonAriaLabel: "no",
    }).then((result) => {
      if (result.isConfirmed) {
        dispatch(tasks.actions.removeAllTasks());
      }
    });
  };
  return (
    <>
      <Button
        type="button"
        className="removeAll"
        onClick={removeItems}
        disabled={clearTasks === 0}
      >
        <h5>
          remove all tasks
          <span role="img" aria-label="bin">
            🗑️
          </span>
        </h5>
      </Button>`

Something like that, this is highly customizable! 😌🧩